utils/publish.sh

```diff
@@ -30,9 +30,9 @@ else
 fi
 
 # Sometimes this is a false alarm and blocks publish
-git checkout yarn.lock
+git checkout pnpm-lock.yaml
```

Comment:
I think we can remove this checkout now since pnpm defaults to frozen lockfile in CI, so it should never happen.

Reply:
lets circle back after PR